coral/docs/frontend-testing.md

```diff
@@ -24,7 +24,7 @@ We use [Jest](hhttps://jestjs.io/docs/) as test runner in Frontend in general an
 
 **⚠️ We use and need this library only for testing React components.**
 
-`Dom testing library` is an opinionated library, which makes it easy to follow a specific way to write test. It’s best used with jest-dom and the custom [jest-dom matchers](https://github.com/testing-library/jest-dom#readme). The combination of both leads to nice readable tests!
+`Dom testing library` is an opinionated library, which supports a specific way of writing test. It is well documented and supports us in having a consistent code base. It’s best used with jest-dom and the custom [jest-dom matchers](https://github.com/testing-library/jest-dom#readme). The combination of both leads to nice readable tests!
```

Comment:
Was this corrected by `alex`?

Reply:
Alex flagged "easy", so I thought about it and rewrote it a bit.

Reply:
Background: I heard the recommendation to [remove potential condescending words](https://dev.to/meeshkan/how-to-remove-condescending-language-from-documentation-4a5p#why) from documentation during a talk a few years back and it resonated a lot with me, because I had to work really hard on removing that kind of words and wording from my language when I worked as a dog trainer 😅 🐶